fix hardcoding of g++ and optimisation flags
fix minor compilation issue "no operator "<<" matches these operands"
author: Kenneth Hoste (HPC-UGent)
diff --git a/VariantMetaCaller_v1.0/makefile.orig b/VariantMetaCaller_v1.0/makefile
index 78ff65d..c49027d 100644
--- a/VariantMetaCaller_v1.0/makefile.orig
+++ b/VariantMetaCaller_v1.0/makefile
@@ -37,7 +37,7 @@ all: VariantMetaCaller
 VariantMetaCaller: $(OBJS) $(USER_OBJS)
 	@echo 'Building target: $@'
 	@echo 'Invoking: GCC C++ Linker'
-	g++ -o"VariantMetaCaller" $(OBJS) $(USER_OBJS) $(LIBS) -fopenmp
+	${CXX} ${CXXFLAGS} ${LDFLAGS} -o"VariantMetaCaller" $(OBJS) $(USER_OBJS) $(LIBS)
 	@echo 'Finished building target: $@'
 	@echo ' '
 
diff --git a/VariantMetaCaller_v1.0/subdir.mk.orig b/VariantMetaCaller_v1.0/subdir.mk
index 530defe..110d8f3 100644
--- a/VariantMetaCaller_v1.0/subdir.mk.orig
+++ b/VariantMetaCaller_v1.0/subdir.mk
@@ -35,7 +35,7 @@ genotypePrioritizer.d
 ./%.o: %.cpp
 	@echo 'Building file: $<'
 	@echo 'Invoking: GCC C++ Compiler'
-	g++ -O0 -g3 -Wall -c -fmessage-length=0 -std=c++11 -fopenmp -MMD -MP -MF"$(@:%.o=%.d)" -MT"$(@:%.o=%.d)" -o"$@" "$<"
+	${CXX} ${CXXFLAGS} -Wall -c -fmessage-length=0 -MMD -MP -MF"$(@:%.o=%.d)" -MT"$(@:%.o=%.d)" -o"$@" "$<"
 	@echo 'Finished building: $<'
 	@echo ' '
 
